samples/bluetooth/peripheral_hids_mouse/README.rst

```diff
@@ -68,6 +68,9 @@ The HID service specification does not require encryption (:kconfig:option:`CONF
 
 Building and running
 ********************
+
+To build this sample with :ref:`lib_nrf_rpc` on `nrf5340dk`, set ``OVERLAY_CONFIG`` to :file:`overlay-nrf_rpc.conf`.
```

Comment:
Could you put console code block with example command call for it under this line?